board.py

```diff
@@ -41,15 +41,15 @@ def check_win(self):
                         return True
         #Major Diagonal check
         for i in range(self.height - 3):
-            for j in range(self.width - 4):
+            for j in range(self.width - 4) # missing last column:
```

Comment:
self.width - 4 gets you columns 0 up to **5** before the end (remember how range works).